habitat/tasks/nav/nav.py

```diff
@@ -731,26 +731,34 @@ def _draw_point(self, position, point_type):
     def _draw_goals_view_points(self, episode):
         if self._config.DRAW_VIEW_POINTS:
             for goal in episode.goals:
-                try:
-                    if goal.view_points is not None:
-                        for view_point in goal.view_points:
-                            self._draw_point(
-                                view_point.agent_state.position,
-                                maps.MAP_VIEW_POINT_INDICATOR,
-                            )
-                except AttributeError:
-                    pass
+                if goal.position[1] > \
```

Comment:
Looks like this pattern is repeated a lot, can you factor that out into a helper function?  Something like 

```python
def is_one_same_floor(pt_source, pt_ref, y_tolerance=2.0):
```

Reply:
Sure, let me change it.